webapp/security/models.py

```diff
@@ -24,64 +26,49 @@
     Column("cve_id", String, ForeignKey("cve.id")),
 )
 
-
-notice_references = Table(
-    "notice_references",
-    Base.metadata,
-    Column("notice_id", String, ForeignKey("notice.id")),
-    Column("reference_id", Integer, ForeignKey("reference.id")),
-)
-
 notice_releases = Table(
     "notice_releases",
     Base.metadata,
     Column("notice_id", String, ForeignKey("notice.id")),
-    Column("release_id", Integer, ForeignKey("release.id")),
-)
-
-cve_bugs = Table(
-    "cve_bugs",
-    Base.metadata,
-    Column("cve_id", String, ForeignKey("cve.id")),
-    Column("bug_id", Integer, ForeignKey("bug.id")),
-)
-
-cve_references = Table(
-    "cve_references",
-    Base.metadata,
-    Column("cve_id", String, ForeignKey("cve.id")),
-    Column("reference_id", Integer, ForeignKey("reference.id")),
-)
-
-cve_packages = Table(
-    "cve_packages",
-    Base.metadata,
-    Column("cve_id", String, ForeignKey("cve.id")),
-    Column("cve_packages", Integer, ForeignKey("package.id")),
+    Column("release_codename", String, ForeignKey("release.codename")),
 )
 
 
 class CVE(Base):
     __tablename__ = "cve"
 
     id = Column(String, primary_key=True)
-    public_date = Column(DateTime)
-    last_updated_date = Column(DateTime)
-    public_date_usn = Column(DateTime)
-    crd = Column(String)
+    published = Column(DateTime)
     description = Column(String)
     ubuntu_description = Column(String)
     notes = Column(JSON)
-    mitigation = Column(String)
-    priority = Column(String)
-    discovered_by = Column(String)
-    assigned_to = Column(String)
-    approved_by = Column(String)
-    cvss = Column(String)  # CVSS vector to convert into Base score
-    references = relationship("Reference", secondary=cve_references)
-    bugs = relationship("Bug", secondary=cve_bugs)
-    packages = relationship("Package", secondary=cve_packages)
+    priority = Column(
+        Enum(
+            "unknown",
+            "negligible",
+            "low",
+            "medium",
+            "high",
+            "critical",
+            name="priorities",
```

Comment:
So another missing priority here is `untriaged` which is one more reason why I didn't want to make this an Enum. So the payload will be invalid

Reply:
Oh I forgot to push the changes to the `ubuntu-cve-tracker` script.

`untriaged` is an internal priority, only relevant to the security team. So I updated the script to turn these into `unknown` instead.